examples/js/GPUComputationRenderer.js

```diff
@@ -99,28 +103,41 @@
 
 function GPUComputationRenderer( sizeX, sizeY, renderer ) {
 
-	this.variables = [];
+	this.renderer = renderer;
 
-	this.currentTextureIndex = 0;
+	this.variables = [];
 
+	// setup the scene for rendering
 	var scene = new THREE.Scene();
 
 	var camera = new THREE.Camera();
 	camera.position.z = 1;
 
+	// setup the initial shader for the mesh
 	var passThruUniforms = {
 		texture: { value: null }
 	};
 
-	var passThruShader = createShaderMaterial( getPassThroughFragmentShader(), passThruUniforms );
+	var passThruShader = createShaderMaterial( getPassThroughFragmentShader(sizeX, sizeY), passThruUniforms );
 
 	var mesh = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2, 2 ), passThruShader );
 	scene.add( mesh );
 
+	this.magFilter = THREE.NearestFilter;
+	this.minFilter = THREE.NearestFilter;
+
+	if ( renderer.extensions.get('OES_texture_float_linear') ) {
```

Comment:
Ah, good point. I wanted the LinearFilter for my fluid solver since I need interpolation when scaling up to the window. Should we make the filter parameter into a user setting?